Core/SubterfugeCore/Core/Entities/Positions/Outpost.cs

```diff
@@ -53,8 +53,16 @@ public abstract class Outpost : IOwnable, ITargetable, IDrillerCarrier, ISubLaun
         /// </summary>
         private SubLauncher _subLauncher;
 
+        /// <summary>
+        /// The position of the outpost in an Rft.
+        /// </summary>
         private RftVector Position;
 
+        /// <summary>
+        /// The players who can see this outpost.
+        /// </summary>
+        private List<Player> _visibleTo;
```

Comment:
I assume this is so that we 'cache' what a player can see and only update it once their vision changes?